lib/open-amp/Kconfig

```diff
@@ -34,3 +34,35 @@ config OPENAMP_RSC_TABLE_IPM_TX_ID
 	help
 	   This option specifies the IPM TX channel ID used in a VRING
 	   for interprocessor communication
+
+config OPENAMP_RSC_TABLE_VENDOR_RSC_LENGTH
+	int "Vendor specific length of the resource"
+	default 0
+	range 0 4
```

Comment:
Question instead of using the size to actually reserve space in the rsc_table, why not create a specific option let's ADD_VENDOR_SPEFICIC_AREA?

I'm afraid that implementors may use for example 0 - sized values and it silently have hard time to debug when this value is badly populated

Having a switch like option enforces the implementor to search the documentation to refer the "right" way to populate the vendor reserved area.

Reply:
Instead of CONFIG_OPENAMP_RSC_TABLE_VENDOR_DATA_FEATURES_0, which means the vendor-specific feature is disable, I can add:
```
config CONFIG_OPENAMP_RSC_TABLE_VENDOR_RSC_FEATURE 
	bool "Vendor specific resource entry"
	depends on OPENAMP_RSC_TABLE
	help
	  Enable vendor-specific features supported by remote processor.
```
Will that be ok?

Reply:
Hi @iuliana-prodan , I like that approach and it sounds more explicit on what this symbol does :)